src/components/Footer/Footer.tsx

```diff
@@ -9,36 +9,36 @@ function Footer() {
     <section className="footer">
       <div className="footer__links">
         <div className="footer__contact">
-          <h3>Contact</h3>
+          <p className="footer__links-header">Contact</p>
```

Comment:
Dlaczego p zamiast h3? Axe tak podpowiedział?

Reply:
Tak, ponieważ przeskoczyło z h1 do h3 od razu, nie było h2 wiec zmieniłem to na p